core/src/main/scala/org/apache/spark/ui/exec/ExecutorsPage.scala

```diff
@@ -58,7 +58,11 @@ private[ui] class ExecutorsPage(
     val diskUsed = storageStatusList.map(_.diskUsed).sum
     val execInfo = for (statusId <- 0 until storageStatusList.size) yield
       ExecutorsPage.getExecInfo(listener, statusId)
-    val execInfoSorted = execInfo.sortBy(_.id)
+    val execInfoSorted = execInfo.filter(x => {
```

Comment:
@srowen 
   val execInfoSorted = execInfo.filter(x => {
      val numPattern = "[0-9]+".r
     val noIntId = numPattern.findFirstIn(x.id).isEmpty
      listener.getExecutorIds.contains(x.id).||(noIntId)
    }).sortBy(_.id)

I intent to filter active executor id. In Executorspage, the type of  executor id is digital or character, eg
![executorspage](https://cloud.githubusercontent.com/assets/9440626/8898172/c012478c-3452-11e5-8312-17f5060339a3.png)
So i use regexes .